build/cloud/README.md

```diff
@@ -7,34 +7,41 @@ operate the cluster with `kubectl`).
 The manifests and scripts in `github.com/kubeedge/kubeedge/build/cloud`
 will be used, so place these files to somewhere you can kubectl with.
 
-### Prepare cloud image
-Ensure your k8s cluster can pull edge controller image. If the
+First, ensure your k8s cluster can pull edge controller image. If the
 image not exist. We can make one, and push to your registry.
 
 ```bash
 cd $GOPATH/src/github.com/kubeedge/kubeedge
 make cloudimage
 ```
 
-### Create secret(Optional)
-For version lower than 1.3.0, we need to generate the tls certs and create
-`06-secret.yaml` based on it.
+Second,Configure the IP address of CloudCore in the field `advertiseAddress`, which will be added to SANs in certificates of CloudCore. For example:
 
-```bash
-cd build/cloud
-../tools/certgen.sh buildSecret | tee ./06-secret.yaml
+```yaml
+modules:
+      cloudHub:
+        advertiseAddress:
+         - 10.10.102.242
 ```
+How to know its IP in advance? You can label a node for running CloudCore and use `nodeSelector`.
```

Comment:
It's a bit confusing that why labeling a node would help get the IP address. It seems the description is incomplete.

Reply:
Just delte this option, func **NewDefaultCloudCoreConfig** will auto fill it with hostip.